exporter/templates/cat1/r3/_2.16.840.1.113883.10.20.24.3.47.xml

```diff
@@ -21,7 +21,7 @@
       </effectiveTime>
     {{end}}
     {{if .EntrySection.AllowedAdministrations}}
-      <repeatNumber value="{{escape .EntrySection.AllowedAdministrations}}" />
+      <repeatNumber value="{{ .EntrySection.AllowedAdministrations }}" />
```

Comment:
For the record, I find the no whitespace style a bit messy in templates. That's the only reason I adjusted it.